versioned/transfer/src/test/java/org/projectnessie/versioned/transfer/AbstractExportImport.java

```diff
@@ -127,7 +127,7 @@ public void manyExportFiles(@TempDir Path targetDir) throws Exception {
           .exportNessieRepository();
     }
 
-    int expectedCommitsFileCount = 126;
+    int expectedCommitsFileCount = 144;
```

Comment:
the total size is bigger due to extra symbols in `CommitMeta` JSON, specifically the `parentCommitHashes` attribute.

Reply:
fix in the latest commit